spec/02-integration/05-proxy/05-dns_spec.lua

```diff
@@ -108,7 +108,7 @@ for _, strategy in helpers.each_strategy() do
 
         local service = bp.services:insert {
           name     = "tests-retries",
-          host     = "nowthisdoesnotexistatall",
+          host     = "nowthisdoesnotexistatall.com",
```

Comment:
probably better to use `.test` instead, see https://en.wikipedia.org/wiki/.test

Reply:
cool, let me try this kind of domain ".test" instead of verifying 500

Reply:
done, and tests passed